stdlib/src/builtin/builtin_list.mojo

```diff
@@ -463,23 +463,30 @@ struct VariadicListMem[
 
 
 # ===----------------------------------------------------------------------===#
-# _LITRefPackHelper
+# VariadicPack
 # ===----------------------------------------------------------------------===#
 
 
 alias _AnyTypeMetaType = __mlir_type[`!lit.anytrait<`, AnyType, `>`]
 
 
-@value
-struct _LITRefPackHelper[
-    is_mutable: Bool, //,
-    lifetime: Lifetime[is_mutable].type,
-    address_space: __mlir_type.index,
```

Comment:
Is this of any significance?

Reply:
Only usage I could find was to do `get_loaded_kgen_pack(self) -> Self._loaded_kgen_pack_type` which is just a subset of what VariadicPack's functionality should be since it is also a `!lit.ref.pack<:variadic` wrapper. That's why I inlined it :man_shrugging:

Reply:
We do want to rely on different address spaces with variadics.  Currently, there's a known bug in the compiler lowering for variadics with non-generic address spaces that we ran into a few months ago internally.  I'd recommend we preserve the address space for now since we do intend to use it.

Of note, I don't think we have any non-generic address space tests (since it mostly comes up with GPU things) and the GPU module is not open source.  We can't easily run those tests in the open source repo right now.

Reply:
Ok, do I just set it up like it was for `_LITRefPackHelper` in its params and its `_mlir_value` ? Kind of a dumb question but I don't really understand MLIR yet, it is a bit too abstract for me sometimes